docs/io/votable/dataorigin.rst

```diff
@@ -0,0 +1,106 @@
+    request_date: 2025-03-05T14:18:05
+    contact: cds-question@unistra.fr
+    publisher: CDS
+
+    ivoid: ivo://cds.vizier/j/aj/167/18
+    citation: doi:10.26093/cds/vizier.51670018
+    reference_url: https://cdsarc.cds.unistra.fr/viz-bin/cat/J/AJ/167/18
+    rights_uri: https://cds.unistra.fr/vizier-org/licences_vizier.html
+    creator: Hong K.
+    ...
+
+Contents and metadata
+---------------------
+
+`astropy.io.votable.dataorigin.extract_data_origin` returns a `astropy.io.votable.dataorigin.DataOrigin` (class) container which is made of:
+
+* a `astropy.io.votable.dataorigin.QueryOrigin` (class) container describing the request.
+  ``QueryOrigin`` is considered to be unique for the whole VOTable.
+  It includes metadata like  the publisher, the contact, date of execution, query, etc.
+
+*  a list of `astropy.io.votable.dataorigin.DatasetOrigin` (class) container for each Element having DataOrigin information.
+   ``DataSetOrigin`` is a basic provenance of the datasets queried. Each attribute is a list.
+   It includes metadata like authors, ivoid, landing pages, ....
+
+Examples
+--------
+
+Get the (Data Center) publisher and the Creator of the dataset
+
+.. code-block:: python
+
+    >>> print(data_origin.query.publisher)  # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    CDS
+    >>> print(data_origin.origin[0].creator)  # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    ['Hong K.']
+
+Other capabilities
+------------------
+DataOrigin container includes VO Elements:
+
+* Extract list of `astropy.io.votable.tree.Info`
+
+.. code-block:: python
+
+    >>> # get DataOrigin with the description of each INFO
+    >>> for dataset_origin in data_origin.origin:  # doctest: +REMOTE_DATA
+    ...    for info in dataset_origin.infos:   # doctest: +REMOTE_DATA
+    ...        print(f"{info.name}: {info.value} ({info.content})")   # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    ivoid: ivo://cds.vizier/j/aj/167/18 (IVOID of underlying data collection)
+    creator: Hong K. (First author or institution)
+    cites: bibcode:2024AJ....167...18H (Article or Data origin sources)
+    editor: Astronomical Journal (AAS) (Editor name (article))
+    original_date: 2024 (Year of the article publication)
+    ...
+
+* Extract tree node `astropy.io.votable.tree.Element`
+
+The following example extracts from header the citation (in APA style).
+
+.. code-block:: python
+
+    # get the Title retrieved in Element
+    >> origin = data_origin.origin[0]  # doctest: +REMOTE_DATA
+    >> vo_elt = origin.get_votable_element()  # doctest: +REMOTE_DATA
+    >> title = vo_elt.description if vo_elt else ""  # doctest: +REMOTE_DATA
+    >> print(f"APA: {','.join(origin.creator)} ({origin.publication_date[0]}). {title} [Dataset]. {data_origin.query.publisher}. {origin.citation[0]}")  # doctest: +REMOTE_DATA +IGNORE_OUTPUT
+    APA: Hong K. (2024-11-06). Period variations of 32 contact binaries (Hong+, 2024) [Dataset]. CDS. doi:10.26093/cds/vizier.51670018
```

Comment:
generic comment: drop the `code-block` directive and use remote-data instead, that would cleanup all the lines a little bit